quipucords/scanner/vcenter/inspect.py

```diff
@@ -208,7 +208,10 @@ def parse_vm_props(self, props, host_dict):
                     facts['vm.cluster'] = host_facts.get('host.cluster')
                     facts['vm.datacenter'] = host_facts.get('host.datacenter')
 
-        vm_name = facts['vm.name']
+        if facts.get('vm.dns_name'):
+            vm_name = facts['vm.dns_name']
+        else:
+            vm_name = facts['vm.name']
```

Comment:
Should you do a `facts.get('vm.name')` here incase vm.name doesn't exist, or is it guaranteed to exist?